.github/workflows/release/Dockerfile.dockerignore

```diff
@@ -0,0 +1,5 @@
+**/.DS_Store
+
+# Allow repeated builds without copying back previous builds
+/releases/*.tar.gz
+/releases/*.tar.gz.sha256sum
```

Comment:
Maybe we could also put others from `.gitignore`?: https://github.com/containerd/containerd/blob/148d21b1ae0718b75718a09ecb307bb874270f59/.gitignore#L4-L13

Reply:
Yes, I was considering that as well, but was a bit on the fence adding "too much" as that may be more of a maintenance nightmare to keep both in sync.

Pro would be better build-caching (they _don't_ affect "git validation" because they're excluded), so ... perhaps? (better caching?).

All that said; using the release Dockerfile somewhat expects to be done from a clean state (which now makes me wonder if `make clean` cleans up those files (it probably _should_).

Reply:
I was wondering if [like buildx repo](https://github.com/docker/buildx/blob/master/.gitignore#L1) for example we could have a common `/bin` folder instead of putting things randomly in root? e.g., `/bin/coverage.txt`, `/bin/releases/`, ...
We could then just need `/bin/` in git and docker ignore?